### PR TITLE
Remove data-view-buffer

### PR DESCRIPTION
**Pull request by filipre**
Removes [`data-view-buffer`](https://www.npmjs.com/package/data-view-buffer) and uses `DataView.prototype.buffer`

### CODE REVIEW

**Comment by filipre** [comment]
Oh wow, looking at https://github.com/thepassle/module-replacements-codemods/pull/38 it seems like `transformInstanceProperty` is what I want to use instead. I can refactor this when #38 is merged.

**Comment by thepassle** [comment]
Would you mind fixing the conflicts in #38? Then we can go ahead and get that merged 🙂

**Comment by filipre** [comment]
@thepassle I rebased on that branch and refactored `data-view-buffer`

File: codemods/index.js [hunk 1]
```diff
@@ -1,5 +1,308 @@
+import isWhitespace from './is-whitespace/index.js';
```
Comment (thepassle) [inline codemods/index.js:1]: Hm, I think something might have gone wrong while rebasing, this file looks different in the main branch: https://github.com/thepassle/module-replacements-codemods/blob/main/codemods/index.js

Comment (filipre) [reply codemods/index.js]: Oops, I fixed it. Thanks for checking

**Comment by thepassle** [comment]
Thanks for the PR!

**Pull request merged**